src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * firstLeg * secondLeg;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: right triangle, area: " + getArea()
```

Comment:
Use only eng in messages/code: There's a typo in 'fisrtLeg', it should be 'firstLeg'.